app/tasks/maintenance.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+global $cli;
+
+require_once __DIR__.'/../init.php';
+
+use Appwrite\Database\Database;
+use Appwrite\Database\Document;
+use Utopia\App;
+use Utopia\CLI\Console;
+
+
+$cli
+    ->task('maintenance')
+    ->desc('Schedules maintenance tasks and publishes them to resque')
+    ->action(function () {
+        // Convert string to integer
+        $interval = App::getEnv('_APP_MAINTENANCE_INTERVAL', '') + 0;
+        //Convert Seconds to microseconds
+        $interval = $interval * 1000000;
+
+        Console::loop(function() {
+            $projects = $consoleDB->getCollection([
```

Comment:
@eldadfux At this point , Im unable to use `$consoleDB`
Let me know how I can access it.

Reply:
You can check for an example here: https://github.com/appwrite/appwrite/blob/d0f7558ddf1c58ecaeffa9503ac84d0ccc11daee/app%2Fworkers%2Fdeletes.php#L189

Note that you'll need the DB env vars available in the container, pass them in the docker-compose file (my previous comment :)).

Reply:
Thank you 😄